packages/mobile/src/components/share-drawer/ShareDrawer.tsx

```diff
@@ -284,23 +418,40 @@ export const ShareDrawer = () => {
   ])
 
   return (
-    <ActionDrawer
-      modalName='Share'
-      rows={getRows()}
-      renderTitle={() => (
-        <View style={styles.title}>
-          <IconShare
-            style={styles.titleIcon}
-            fill={neutral}
-            height={18}
-            width={20}
+    <>
+      {shouldRenderShareToStorySticker ? (
+        <ViewShot
+          style={styles.viewShot}
+          ref={viewShotRef}
+          options={{ format: 'png' }}
+        >
+          <ShareToStorySticker
+            onLoad={handleShareToStoryStickerLoad}
+            track={content.track}
+            // TODO(nkang): Not sure if this helps or hurts chances of successful image:
+            // user={account}
```

Comment:
Shouldn't need the `user`, the image hook will use the one in cache. This was only needed for search items where the user wasn't in the cache already